auth_app/models/users.py

```diff
@@ -1,16 +1,28 @@
 import uuid
+from enum import Enum
 
-from sqlalchemy import Boolean, Column, MetaData, String, Table
+from sqlalchemy import Boolean, Column
+from sqlalchemy import Enum as Enum_Sql
+from sqlalchemy import String, Table
 from sqlalchemy.dialects.postgresql import UUID
 
-metadata_obj = MetaData()
+from auth_app.models.meta import metadata_obj
+
+
+class UserRole(str, Enum):
+    ADMIN = 'ADMIN'
+    USER = 'USER'
+    STAFFER = 'STAFFER'
+    OTHER = 'OTHER'
+
 
 users = Table(
     "users",
     metadata_obj,
     Column("id", UUID(as_uuid=True), primary_key=True, default=uuid.uuid4),
     Column("email", String, unique=True, nullable=False),
     Column("password_hash", String, nullable=False),
+    Column("role", Enum_Sql(UserRole), nullable=False, default=UserRole.USER),
     Column("is_verified", Boolean, default=False),
     Column("is_active", Boolean, default=True),
 )
```

Comment:
Ты объявляешь таблицы в декларативном стиле. Это иногда окей, но вообще почти никогда так не делают, потому что с этим крайне неудобно работать
Надо будет заменить на такой вариант
```suggestion
class User(Base):
    __tablename__ = "users"

    id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
    email = Column(String, unique=True, nullable=False)
    password_hash = Column(String, nullable=False)
    role = Column(Enum_Sql(UserRole), nullable=False, default=UserRole.USER)
    is_verified = Column(Boolean, default=False)
    is_active = Column(Boolean, default=True)

    refresh_tokens = relationship("RefreshToken", back_populates="user")
```
примерно так

Reply:
исправил